### PR TITLE
feat: add `ibc` and `ibc-transfer` modules

### PR DESCRIPTION
**Pull request by lumtis**
# Description

Closes: https://github.com/zeta-chain/node/issues/1510

### CODE REVIEW

**Review by kingpinXD — APPROVED** [review]
lgtm , do we need a migration script ? 
If so would that be in a separate pr ?

**Comment by lumtis** [comment]
> lgtm , do we need a migration script ? If so would that be in a separate pr ?

Yes, will need but need adding IBC chain testing in E2E tests for this, which would be in a later PR

File: app/app.go [hunk 1]
```diff
@@ -550,6 +683,7 @@ func New(
 
 	app.mm.SetOrderBeginBlockers(
 		upgradetypes.ModuleName,
+		capabilitytypes.ModuleName,
```
[the remark this answers is not in the document]
Comment (lumtis) [reply app/app.go:686]: Yes, it might make more sense

**Comment by lumtis** [comment]
Seems there is some issues with the upgrade test

**Pull request merged**